integration/Jenkinsfiles/ftest.groovy

```diff
@@ -0,0 +1,177 @@
+                                            sh """#!/bin/bash -ex
+                                                  ! grep -E '^[0-9]{4}-[0-9]{2}-[0-9]{2}.*ERROR.*' nuxeo-distribution/nuxeo-server-funkload-tests/target/tomcat/log/server.log
+                                            """
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                "webdriver" : {
+                    node('SLAVE') {
+                        stage('webdriver') {
+                            ws("$WORKSPACE-webdriver") {
+                                unstash "clone"
+                                timeout(time: 2, unit: 'HOURS') {
+                                    withBuildStatus("$DBPROFILE-$DBVERSION/ftest/cmis", sha) {
+                                        withDockerCompose("$JOB_NAME-$BUILD_NUMBER-webdriver", "integration/Jenkinsfiles/docker-compose-$DBPROFILE-${DBVERSION}.yml", "mvn -B -f $WORKSPACE/nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/pom.xml clean verify -Pqa,tomcat,$DBPROFILE") {
+                                            archive 'nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/failsafe-reports/*, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/*.png, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/*.json, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/*.log, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/log/*, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/**/nxserver/config/distribution.properties, nuxeo-distribution/nuxeo-server-cmis-tests/target/nxtools-reports/*, nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/results/*/*'
+                                            junit '**/target/surefire-reports/*.xml, **/target/failsafe-reports/*.xml, **/target/failsafe-reports/**/*.xml'
+                                            sh """#!/bin/bash -ex
+                                                  ! grep -E '^[0-9]{4}-[0-9]{2}-[0-9]{2}.*ERROR.*' nuxeo-distribution/nuxeo-jsf-ui-webdriver-tests/target/tomcat/log/server.log
+                                            """
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            )
+        }
+    }
+}
+
+// should be in shared library
```

Comment:
Maybe we could make the move right now to https://github.com/nuxeo/nuxeo-pipeline-library rather than postponing it with no identified JIRA ?